src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/CondEquipCompDispensable.java

```diff
@@ -0,0 +1,71 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+@Name("Equippable Component - Can Be Dispensed")
+@Description("Whether an item can be dispensed by a dispenser. "
+	+ "Note that equippable component elements are experimental making them subject to change and may not work as intended.")
+@Example("""
+	if {_item} can be dispensed:
+		add "Dispensable" to lore of {_item}
+	""")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	if {_component} is not able to be dispensed:
+		allow {_component} to be dispensed
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class CondEquipCompDispensable extends PropertyCondition<EquippableWrapper> implements EquippableExperiment {
+
+	static {
+		Skript.registerCondition(CondEquipCompDispensable.class, ConditionType.PROPERTY,
+			"%equippablecomponents% ((is|are) able to|can) be dispensed",
+			"%equippablecomponents% ((is not|isn't|are not|aren't) able to|(can not|can't)) be dispensed"
```

Comment:
I would use the `getPatterns` method instead, twice:
```java
PropertyCondition.getPatterns(PropertyType.BE, "(able to be dispensed|dispensable)", "equippablecomponents");
PropertyCondition.getPatterns(PropertyType.CAN, "be dispensed", "equippablecomponents");
```
Easier to read, improves consistency, and produces better patterns for docs and users